client/lib/cart-values/index.js

```diff
@@ -184,6 +187,15 @@ function paymentMethodName( method ) {
 		'brazil-tef': 'Transferência bancária',
 	};
 
+	// Temporarily override 'credit or debit' with just 'credit' for india
+	// while debit cards are served by the paywall
+	if ( method === 'credit-card' ) {
+		const userCountryCode = requestGeoLocation().data;
+		if ( 'IN' === userCountryCode ) {
+			return i18n.translate( 'Credit card' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **43** times:
`translate( 'Credit Card' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/support-for-hosted-paywall)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).